test/fixtures/runner.js

```diff
@@ -0,0 +1,79 @@
+
+const path = require('path')
+const fs = require('fs')
+const t = require('tap')
+
+/**
+ * @param {object} option
+ * @param {boolean} option.snapshot true (will update) or false (will do nothing)
+ * @param {string} option.file the basename of the cache file
+ */
+const runner = (option, examples, tests) => {
+  if (!option.file) {
+    throw new Error('Cache file must be set')
+  }
+  let SNAP = option.snapshot
+  const snpashot = path.join(__dirname, '../snapshots', option.file)
```

Comment:
Would it make sense to use tap's builtin snapshot management here?

Reply:
Yes fixed